integration/kubernetes/init.sh

```diff
@@ -109,6 +111,14 @@ trap 'sudo -E sh -c "rm -r "${kubeadm_config_file}""' EXIT
 if [ "${BAREMETAL}" == true ] && [[ $(wc -l /proc/swaps | awk '{print $1}') -gt 1 ]]; then
 	sudo swapoff -a || true
 fi
+
+#deep cleanup k8s in arm64 environment
+if [ "${arch}" == "arm64" ]; then
+	info "reinstall kubeadm kubelet before initialize k8s"
+	sudo apt purge kubelet -y
+	sudo apt install kubelet kubeadm -y
```

Comment:
Oi!

This is quite error prone.  We shouldn't be uninstalling kubelet (and all its dependencies) here and then installing it again, and if we do, we should be ensuring the version to be installed, as done in: https://github.com/kata-containers/tests/blob/main/.ci/install_kubernetes.sh

I wonder why the cleanup functions (https://github.com/kata-containers/tests/blob/main/.ci/aarch64  /clean_up_aarch64.sh) are not working properly and I wonder whether patching it there would be a better solution.

Reply:
I try to cleanup the host hard, but it doesn't work. I have no other way than reinstall the kubelet. can I reinstall kubelet of the right version here?

Reply:
What about:
1. Check whether we have something installed and, if we do, we purge it as part of https://github.com/kata-containers/tests/blob/main/.ci/install_kubernetes.sh ... that would be the very first step before actually installing something;
2. We just call that script here (as that script already takes care of using the correct versions and whatnot);

IMHO, we could even do it independently of the architecture.

Does this sound reasonable?

Reply:
Nice! thanks @fidencio, let me try it.